index.html

```diff
@@ -1093,49 +1095,21 @@
 cookies and other storage (which is sometimes automatic at the end of each
 session).
 
-Even though this is the default, [=user agents=] can further restrict their
-definition of [=machine-enforceable context=] to better aid their users for
-instance by partitioning identities per subdomain or site path when the
-structure of a given site is known.
-
-Where possible, [=user agents=] should prevent people from being
-[=cross-context recognition|recognized=] across [=partitions=] unless they
-intend to be recognized.
-
-If a [=person=] visits two or more web pages from different [=partitions=],
-and does not explicitly express the same identity on each visit, the user agent should
-take steps to prevent [=same-site recognition=]. Note that sites can do harm
-even if they can't be *completely* certain that visits come from the same
-person. Further, note that sites must not assume that a person intends to be
-recognised across different sites simply because they are using the same
-identifier (such as an email address) with both or are relying on the same
-single sign-on identity.
-
-[=User agents=] and identity-related web features can do the following to help
-people present the [=identity=] that they want to present to a given website:
-
-* Do not reveal someone's profile information without their consent.
-* Do not leak someone's profiles, including which types of profiles they have,
-without their consent.
-* Make it clear to the user which identity is being presented on a given
-website.
-* Disclose the least amount of identifying information, e.g.:
-  * Only share the parts of someone's profile information that is needed by
-  the website.
-  * Minimize the shared data; for example if a website requires age for
-  verification, then share that the
-  user is older than the minimum age, as opposed to specific age or date of
-  birth.
-* Support the user's intent to present a directed identity or a site-specific
-identity. E.g., if the website requires an email, then the user may choose a
-directed identifier without disclosing their real email.
-
-This principle is limited in cases where preventing
-[=cross-context recognition|recognition=] would break fundamental aspects of
-the web. In many cases it's possible to change the design in a way that avoids
-the violation without breaking valid use cases, but for cases where that's not
-possible we refer you to [[[Privacy-Threat]]] which discusses tradeoffs in
-detail.
+When a [=user agent=] does have better information, it should adjust its
+[=partitions=] accordingly, for instance by partitioning identities per
+subdomain or site path when the structure of a given site is known.
```

Comment:
If I've captured Pete's suggestion correctly, we agree that

```suggestion
When a [=user agent=] knows that a site includes multiple contexts, it
should adjust its [=partitions=] accordingly, for instance by partitioning
identities per subdomain or site path.
```

works for everyone.